og_ui/og_ui.routing.yml

```diff
@@ -15,3 +15,29 @@ og_ui.settings:
     _title: 'OG settings'
   requirements:
     _permission: 'administer group'
+
+og_ui.roles_permissions_overview:
+  path: 'admin/config/group/{type}'
+  defaults:
+    _controller: '\Drupal\og_ui\Controller\OgUiController::rolesPermissionsOverviewPage'
+    _title_callback: '\Drupal\og_ui\Controller\OgUiController::rolesPermissionsOverviewTitleCallback'
+    type: 'roles'
+  requirements:
+    _permission: 'administer group'
+    type: '^(roles|permissions)$'
+
+og_ui.roles_form:
+  path: 'admin/config/group/roles/{entity_type}/{bundle}'
+  defaults:
+    _form: '\Drupal\og_ui\Form\OgRolesForm'
+    _title: '@todo - create title callback'
```

Comment:
No that title callback is for the `og_ui.roles_permissions_overview`. I needed to add the two other routes (`og_ui.roles_form` and `og_ui.permissions_form`) so that I could generate the links in the overview, but these two forms are not implemented yet. I have added that @todo there as a reminder that these forms need a title callback :)